### ISSUE TITLE
Bandwidth doesn't work with calico 3.2.4

### ISSUE THREAD

**Issue by CallMeFoxie** [containernetworking/plugins#242]
Hi,
I've been testing the bandwidth plugin and it seems to have a problem with calico 3.2.4 with this config:
```
{
  "cniVersion": "0.3.0",
  "plugins": [
  {
      "etcd_endpoints": "https://etcd_redacted_servers",
      "etcd_key_file": "/www/adm/calico/cert/etcd-key.pem",
      "kubernetes": {
        "kubeconfig": "/www/adm/kubernetes/conf/kubeconfig.kubelet.yaml"
      },
      "log_level": "info",
      "ipam": {
        "type": "host-local",
        "subnet": "usePodCidr"
      },
      "etcd_cert_file": "/www/adm/calico/cert/etcd-cert.pem",
      "policy": {
        "type": "k8s"
      },
      "type": "calico",
      "etcd_ca_cert_file": "/www/adm/calico/cert/etcd-ca.pem"
    },
    {
      "snat": true,
      "capabilities": {
        "portMappings": true
      },
      "type": "portmap"
    },
    {
      "type": "bandwidth",
      "capabilities": {"bandwidth": true}
    }
  ],
  "name": "mynet"
}
```

I've tracked it down to this line:

https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/main.go#L145

where the `iface.Sandbox = eth0` in this case and never matches any interface.

Is it problem on calico side, my configuration side or bandwidth plugin side? When I comment out that check it works fine.

**Comment by squeed** [comment]
Nice find!

It looks like a Calico bug. Specifically, the spec says that plugins should not set "iface.Sandbox" if the interface is in the "root" namespace. @matthewdupre  - any thoughts?

**Comment by CallMeFoxie** [comment]
Oh it seems to have been fixed in the 3.3 branch :) 

https://github.com/projectcalico/cni-plugin/pull/607

**Comment by dcbw** [comment]
thanks for the update, closing.
